### PR TITLE
2015-06-21_jonathan_max_cohort_2_planning in meetings

### PR DESCRIPTION
**Pull request by MaxWofford**

### CODE REVIEW

File: meetings/2015-06-21_jonathan_max_cohort_2_planning.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Attendance: Jonathan, Max
+
+Plan for Sunday (2015-06-21):
+  1. Edit facilitator module for day 1
+  1. Write out student modules for day 1
+  1. Create a way to measure metrics for day 1 (Reach out to @andreadeng)
+    * For each module set objectives without extracting tests
+  1. Write out the rest of facilitator modules for week 1
```
Comment (zachlatta) [inline meetings/2015-06-21_jonathan_max_cohort_2_planning.md]: Is this markdown correct?

**Comment by zachlatta** [comment]
Is the date format in the file name correct?

**Comment by zachlatta** [comment]
@jonleung in the future can you make sure comments are addressed before merging a pull request? I went ahead and opened #61 to fix the naming.

**Comment by jonleung** [comment]
@zachlatta :+1:

**Pull request merged**